tee-worker/ts-tests/common/call.ts

```diff
@@ -1,25 +1,24 @@
 import { ApiPromise } from '@polkadot/api';
 import { Metadata, TypeRegistry } from '@polkadot/types';
 import WebSocketAsPromised from 'websocket-as-promised';
-import { WorkerRpcReturnValue, IdentityContext } from './type-definitions';
 import { HexString } from '@polkadot/util/types';
-
+import { RequestBody, WorkerRpcReturnValue } from '../common/type-definitions';
 //rpc call
 export async function sendRequest(
     wsClient: WebSocketAsPromised,
-    request: any,
+    request: RequestBody,
     api: ApiPromise
 ): Promise<WorkerRpcReturnValue> {
     const resp = await wsClient.sendRequest(request, { requestId: 1, timeout: 6000 });
-    const resp_json = api.createType('WorkerRpcReturnValue', resp.result).toJSON() as WorkerRpcReturnValue;
```

Comment:
Wondering, shouldn't we validate the payload here to make sure it conforms to the type? :thinking:

Reply:
I modified the usage of WorkerRpcReturnValue, it seems that we still need to manually define it here, as using the generated one is not good enough.

Reply:
btw;Is it better to have this comment in the modified code? Otherwise, it might be confusing to read this part directly.Just a small friendly reminder.:)

Reply:
This comment was prompted by the `as WorkerRpcReturnValue`, so I put it on that line :D